public_html/wp-content/plugins/camptix/camptix.php

```diff
@@ -3390,7 +3397,10 @@ public function get_segment( $relation, $conditions ) {
 		);
 
 		foreach ( $conditions as $condition ) {
-			if ( empty( $condition['field'] ) || empty( $condition['op'] ) || ! isset( $condition['value'] ) )
+			if ( empty( $condition['field'] ) || empty( $condition['op'] ) )
+				continue;
+
+			if ( "attended" != $condition['field'] && ! isset( $condition['value'] ) )
```

Comment:
Some PHP code style nitpicks: this should use single quotes around `attended`, and use strict comparisons: 
```suggestion
			if ( 'attended' !== $condition['field'] && ! isset( $condition['value'] ) )
```